engine/src/main/scala/cromwell/server/CromwellServer.scala

```diff
@@ -43,9 +41,14 @@ class CromwellServerActor(cromwellSystem: CromwellSystem)(override implicit val
   val interface = webserviceConf.getString("interface")
   val port = webserviceConf.getInt("port")
 
-  val routeUnwrapped: Boolean = cromwellSystem.conf.as[Option[Boolean]]("api.routeUnwrapped").getOrElse(false)
-
-  val allRoutes: Route = routes.wrapped("api", routeUnwrapped) ~ swaggerUiResourceRoute
+  /**
+    * /api routes have special meaning to devops' proxy servers. NOTE: the oauth mentioned on the /api endpoints in
+    * cromwell.yaml is broken unless the swagger index.html is patched. Copy/paste the code from rawls or cromiam if
+    * actual cromwell+swagger+oauth+/api support is needed.
+    */
+  val apiRoutes: Route = pathPrefix("api")(concat(workflowRoutes))
```

Comment:
I'm guessing it's not really feasible to do it other than this piecemeal fashion?

One thing I really came to loathe (and thus undid w/ the akka http transition) was having to both define the route and then make sure it gets added to the aggregation. 

This isn't nearly as egregious as presumably things will fall into one of the two categories, but was curious if this was more a matter of necessity or taste.